cfgov/unprocessed/js/modules/Analytics.js

```diff
@@ -66,11 +65,11 @@ var Analytics = {
    * @param {object} dataLayerOptions Type of event.
    */
   sendEvent: function( dataLayerOptions ) {
-    var callback = dataLayerOptions.eventCallback;
+    const callback = dataLayerOptions.eventCallback;
     if ( Analytics.tagManagerIsLoaded ) {
       window.dataLayer.push( dataLayerOptions );
     } else if ( callback && typeof callback === 'function' ) {
-      callback();  // eslint-disable-line callback-return, inline-comments, max-len
+      callback(); // eslint-disable-line callback-return, inline-comments, max-len
```

Comment:
How do you feel about moving the disable callback to the line above so we don't have to include the other two?

Reply:
👍  I like and we should do this everywhere that we have unnecessary ignore rules.